crates/bifrost/src/providers/replicated_loglet/tasks/digests.rs

```diff
@@ -182,7 +182,7 @@ impl Digests {
                 known_copies,
                 &mut thread_rng(),
                 &networking.metadata().nodes_config_ref(),
-                &NodeSet::empty(),
+                &NodeSet::new(),
```

Comment:
why not 
```suggestion
                &NodeSet::default(),
```

Reply:
No strong opinion here. I'll change it to default.